sections/04_study.md

```diff
@@ -640,8 +640,11 @@ variant and fed these vectors into a fully connected deep neural network
 [@tag:Torracinta2016_deep_snp]. Unfortunately, this feature set required at
 least 15 iterations of software development to fine-tune, which will likely not
 be generalizable. Going forward, we foresee that variant calling will benefit
-most from optimized neural network architectures and better methods of encoding
-raw sequence and quality data (e.g. in tensor rather than RGBA format).
+more from optimizing neural network architectures than from developing features
+by hand. An interesting and informative next step would be to test whether
```

Comment:
I agree with making the point about leveraging pretrained networks explicit. Moved some words around and added a reference to Xception